docs/tutorials/working-with-webassembly.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Working with WebAssembly
+---
+
+<head>
+  <link rel="canonical" href="https://docs.rancherdesktop.io/tutorials/working-with-webassembly"/>
+</head>
+
+Rancher Desktop 1.13.0 added experimental support for running WebAssembly (WASM) applications. This feature needs to be enabled in  [Preferences > Container Engine > General](../ui/preferences/container-engine/general.md).
+
+:::caution warning
+Note that when using the `moby` container engine, enabling the WASM feature switches to a different image store, so previously built or downloaded images will not be available and must be built or downloaded again. The images are not lost; Rancher Desktop will switch back to the old image store when WASM is disabled again.
+:::
+
+## Managing containerd WASM shims
+
+To run WebAssembly applications on a container runtime requires a "shim" for each WASM runtime/framework that is going to be used.
+
+Rancher Desktop 1.13 comes bundled with a the `containerd-spin-shim-v2` shim preinstalled. Future releases are expected to download additional shims automatically when the feature is enabled.
+
+For now additional shims can be installed by the user into the `containerd-shims` cache directory on the host. The location is
```

Comment:
```suggestion
However, you can install additional shims by manually downloading and copying the shim binaries to the `containerd-shims` cache directory on the host. The location is
```

Reply:
The whole document is written in passive voice (which I found a bit painful), so adding a "you can do this" sentence feels out of place. If we want to change the tone, then the whole page should be adjusted accordingly.